### PR TITLE
generator for spectr.yml

### PR DESCRIPTION
**Pull request by gaurav-singh1990**
Generator for spectr.yml

### CODE REVIEW

File: lib/generators/spectr/USAGE [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Description:
+    Create spectr.yml in Rail.root/config folder which will be user by this gem to add database details
+
+Example:
+    rails generate spectr Thing
```
Comment (kapilbhosale) [inline lib/generators/spectr/USAGE]: Why a Named generator is needed.

Comment (gaurav-singh1990) [reply lib/generators/spectr/USAGE]: In future if we need to pass arguments from command line then we will need it

File: lib/generators/spectr/spectr_generator.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+require 'rails/generators/base'
+
+
+class SpectrGenerator < Rails::Generators::NamedBase
+  source_root File.expand_path('../templates', __FILE__)
+
+  def create_spectr_file
```
[the remark this answers is not in the document]
Comment (gaurav-singh1990) [reply lib/generators/spectr/spectr_generator.rb]: formatting of file becomes weird if I use above code.

[the remark this answers is not in the document]
Comment (gaurav-singh1990) [reply lib/generators/spectr/spectr_generator.rb]: won't change

File: lib/generators/spectr/USAGE [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Description:
+    Create spectr.yml in Rail.root/config folder which will be user by this gem to add database details
```
Comment (akshaymohite) [inline lib/generators/spectr/USAGE]: Typo `used by`

**Pull request merged**